packages/quantic/scripts/build/util/sfdx.ts

```diff
@@ -26,18 +26,20 @@ export function sfdx<T = SfdxResponse>(command: string): Promise<T> {
         maxBuffer: 1024 * 1024 * 1.5,
       },
       (error, stdout) => {
-        if (error) {
-          let jsonOutput;
+        let jsonOutput: unknown;
+        if (stdout) {
           try {
             jsonOutput = JSON.parse(strip(stdout));
-          } catch (e) {
+            if (error) {
+              reject(jsonOutput || error);
```

Comment:
maybe we should throw here instead so that the stdout gets logged on line 39

Reply:
or just add logging here as well given we reject here with different info

Reply:
added